doc/whatsnew/2.9.rst

```diff
@@ -62,6 +62,9 @@ New checkers
 Other Changes
 =============
 
+
```

Comment:
I think this should be in 2.10.rst.

Reply:
No problem, I'll put it under 2.10 as soon as I figured out what's the problem with the implementation under Python 3.6.

Reply:
Sorry I made a mistake here, we're going to release it in 2.9.4, but I thought ``consider-using-with`` was added in 2.9 (so it would not make sense to have a new checker and a bug fix for it in the same what's new), but in fact it's been added in 2.8. Could you revert this commit, please?